protocol/message.go

```diff
@@ -90,6 +91,18 @@ type MonitoringRequest struct {
 	EndEpoch   uint64 `json:"end_epoch"`
 }
 
+// An AuditingRequest is a message with a CONIKS key directory's address
+// as a string and an epoch as a uint64 that a CONIKS client sends to
+// a CONIKS auditor to request the given directory's STR at the given
+// epoch.
+//
+// The response to a successful request is an ObservedSTRs with
+// a list of STRs covering the epoch range [Epoch, d.LatestSTR().Epoch].
+type AuditingRequest struct {
+	DirectoryAddr string `json:"directory_addr"`
```

Comment:
Thanks for the reminder. I'm in the process of rebasing onto master, so I'll remove the tags as part of that.